### PR TITLE
Re-formulate docker instruction

### PR DESCRIPTION
**Pull request by cansav09**
This PR mainly addresses #43, but also includes 
- flash drive instructions add (#44) and
- [hello world issue](https://github.com/AlexsLemonade/training-admin/issues/41) instructions. 

The parallel changes for this hello world issue are not merged yet (still working on the indicator image) but they are on the `training-modules` branch `cansav09/hello-world` on [PR #115](https://github.com/AlexsLemonade/training-modules/pull/115) if you'd like to see that in conjunction.

### CODE REVIEW

**Comment by jaclyn-taroni** [comment]
If the internet is sufficiently fast, we will not have people get the Docker image from the flashdrives = the flashdrive instructions should not tell folks to get the Docker image from the flashdrive. 

I would change this https://github.com/AlexsLemonade/RNA-Seq-Exercises/pull/45/files#diff-1a41190f8719933a49722e818e570f28R12 to basically say get the `tar.gz` file from the flashdrive instead of linking to a different set of instructions.

**Comment by cansav09** [comment]
> flashdrives = the flashdrive instructions should not tell folks to get the Docker image from the flashdrive.
I had thought about that but then thought that it wouldn't hurt for them to copy it anyway while they have the flash drive as opposed to accidentally passing the flashdrive on and then realizing they still need it when they get to that step.

File: docker-load.md [hunk 1]
```diff
@@ -84,7 +91,77 @@ You should see output like:
 
 ```
 REPOSITORY                         TAG                 IMAGE ID            CREATED             SIZE
-ccdl/training_rnaseq               2019-chicago        22f2b4f05051        3 days ago          5.28GB
+ccdl/training_rnaseq               2019-czi        22f2b4f05051        3 days ago          5.28GB
```
Comment (jaclyn-taroni) [inline docker-load.md]: Can you remove this image id in a way that indicates it needs to be updated by us at a later point?

**Comment by jaclyn-taroni** [comment]
Gotcha, yea that's fair. I still see linking out to a different set of instructions as potentially confusing.

**Comment by cansav09** [comment]
> Gotcha, yea that's fair. I still see linking out to a different set of instructions as potentially confusing.

The way I see it is, the step 1 on `docker-load.md` that you've referenced is just them double checking that they have that file and have followed the flashdrive instructions. They should have already done that step. What may potentially be confusing is why they copy it if they use `docker-pull.md` and we never reference that file again. IDK. Perhaps @dvenprasad has thoughts on this matter?

File: docker-pull.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+2. Run the container. Change the `<PASSWORD>` in the line below to whatever you'd
+  like.
+```
+docker run -e PASSWORD=<PASSWORD> -p 8787:8787 ccdl/training_rnaseq:2019-czi
+```
+
+3. Open `Kitematic` - you should see an image running.
+
+4. `Settings` > `Volumes` > Set local folder to `training-modules` that was
+transferred from the flash/hard drive, using the `CHANGE` button.
```
Comment (jaclyn-taroni) [inline docker-pull.md]:
```suggestion
transferred from the flash drive, using the `CHANGE` button.
```
might as well keep it consistent

File: docker-load.md [hunk 3]
```diff
@@ -84,7 +91,77 @@ You should see output like:
...
+  like. Make sure to get rid of `<` and `>`. Also note that your chosen PASSWORD
+  cannot have a `$`.
+```
+docker run -e PASSWORD=<PASSWORD> -p 8787:8787 ccdl/training_rnaseq:2019-czi
+```
+
+2. Open `Kitematic` - you should see an image running.
+
+3. `Settings` > `Volumes` > Set local folder to `training-modules` that was
+transferred from the flash/hard drive, using the `CHANGE` button.
```
Comment (jaclyn-taroni) [inline docker-load.md]:
```suggestion
transferred from the flash drive, using the `CHANGE` button.
```

File: README.md [hunk 4]
```diff
@@ -2,73 +2,15 @@
...
-- In Windows, go to `Command Prompt`, type: `ipconfig` and click enter.
-  Find the number that corresponds to the `Virtual Box Host Network` and the
-  `IPv4 Address`. Copy and paste it.
-  Put that number and `:8787` at the end of it in your browser.
-
-Resume with step 6 and 7.
+This module focuses on the analysis of RNA-seq data. To set up your Docker
+RNA-seq environment, you need to follow the instructions specified in the [schedule](schedule.md)
+under "Getting Started". Setting up this environment will
+provide you with all the software and packages you need for the [RNA-seq workshop modules](https://github.com/AlexsLemonade/training-modules).
```
Comment (jaclyn-taroni) [inline README.md]: I would take out this link to the whole repository for future-proofing purposes.

File: docker-load.md [hunk 5]
```diff
@@ -1,42 +1,49 @@
...
 
-Copy `ccdl_training_rnaseq.tar.gz` from the flash drives we have distributed to your Desktop folder
+1. Be sure that you have copied `ccdl_training_rnaseq.tar.gz` from the flash drive
+as was instructed in [these steps](flashdrive-instructions.md). If you do not
+have this file yet, follow [those instructions](flashdrive-instructions.md) before
+attempting to complete these next steps.
 
 ### Next steps
 
 * [Mac OS X and Linux](#mac-os-x-and-linux)
```
Comment (jaclyn-taroni) [inline docker-load.md]:
```suggestion
* [Mac OS X and Linux](#next-steps-for-mac-os-x-and-linux-users)
```

File: docker-load.md [hunk 6]
```diff
@@ -1,42 +1,49 @@
...
 * [Windows 10 Pro](#windows-10-pro)
```
Comment (jaclyn-taroni) [inline docker-load.md]:
```suggestion
* [Windows 10 Pro](#next-steps-for-windows-10-pro-users)
```

File: docker-load.md [hunk 7]
```diff
@@ -84,7 +91,77 @@ You should see output like:
...
+
+8. Click `Enter`. You should receive an indication that you are successfully set
+up. Keep this indicator handy for the instructors and helpers as they come
+around to check.
+
+### If Kitematic doesn't work:
+
+If all else fails and Kitematic is not working for you, go to your `Terminal` or
+`Command Prompt` (for Mac or Windows respectively) and type in the following, but
+replacing <PATH_TO_TRAINING_FOLDERS> bit with the absolute path to
```
Comment (jaclyn-taroni) [inline docker-load.md]:
```suggestion
replace <PATH_TO_TRAINING_FOLDERS> with the absolute path to
```

File: docker-load.md [hunk 8]
```diff
@@ -84,7 +91,77 @@ You should see output like:
...
+"main directory" that was transferred from the flash/hard drive.
```
Comment (jaclyn-taroni) [inline docker-load.md]:
```suggestion
`training-modules` that was transferred from the flash drive.
```

File: docker-load.md [hunk 9]
```diff
@@ -84,7 +91,77 @@ You should see output like:
...
+```
+After starting your container this way, you can get to the RStudio window in
+a similar way as described above:
+- In Mac, type: `localhost:8787` in your web browser.
+- In Windows, go to `Command Prompt`, type: `ipconfig` and click enter.
+  Find the number that corresponds to the `Virtual Box Host Network` and the
+  `IPv4 Address`. Copy and paste it.
+  Put that number and `:8787` at the end of it in your browser.
+
+Resume with step 6 and 7.
```
Comment (jaclyn-taroni) [inline docker-load.md]:
```suggestion
Resume with step 5 of part 2.
```
I think?

File: docker-load.md (hunk 8 above)
Comment (jaclyn-taroni) [reply docker-load.md]: Carry over these changes to the other set of instructions as well.

File: README.md [hunk 10]
```diff
@@ -2,73 +2,15 @@
...
+
+Briefly, you need to have successfully installed [Docker](https://github.com/AlexsLemonade/training-modules/blob/master/docker-install/README.md). Then you need to [obtain the workshop data from the flash drives.](flashdrive-instructions.md)
+
+Lastly, there are two different sets of instructions, for setting up your RNA-seq Docker container,
```
Comment (jaclyn-taroni) [inline README.md]:
```suggestion
Lastly, there are two different sets of instructions for setting up your RNA-seq Docker container:
```

**Comment by jaclyn-taroni** [comment]
Feel free to leave as is. The compressed Docker image should be less than 3 GB.

File: docker-load.md (hunk 1 above)
Comment (cansav09) [reply docker-load.md]: Yes. I will do that.

File: docker-pull.md (hunk 2 above)
Comment (cansav09) [reply docker-pull.md]: Yes.

File: docker-load.md (hunk 9 above)
Comment (cansav09) [reply docker-load.md]: Looks like Step 4 of part 2 but point taken.  I think the other one is also wrong in this respect.

Comment (cansav09) [reply docker-load.md]: Nope, never mind. You and past Candace were right.

**Pull request merged**